From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: selftests/fpu: add missing MODULE_DESCRIPTION() macro
Date: Sat, 22 Jun 2024 07:55:05 -0700

make allmodconfig && make W=1 C=1 now reports:
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/test_fpu.o

Add the missing invocation of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240622-md-i386-lib-test_fpu_glue-v1-1-a4e40b7b1264@quicinc.com
Fixes: 9613736d852d ("selftests/fpu: move FP code to a separate translation unit")
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/test_fpu_glue.c |    1 +
 1 file changed, 1 insertion(+)

--- a/lib/test_fpu_glue.c~selftests-fpu-add-missing-module_description-macro
+++ a/lib/test_fpu_glue.c
@@ -59,4 +59,5 @@ static void __exit test_fpu_exit(void)
 module_init(test_fpu_init);
 module_exit(test_fpu_exit);
 
+MODULE_DESCRIPTION("Test cases for floating point operations");
 MODULE_LICENSE("GPL");
_
